changelog.md

```diff
@@ -8,6 +8,9 @@
   - Added `BpkBackgroundImage` component which can be used to create a container with a background image
   - Added `withLazyLoading` HOC which allows `BpkImage` or `BpkBackgroundImage` to be loaded only when in view
   - Added `withLoadingBehavior` HOC which allows `BpkImage` or `BpkBackgroundImage` to be shown only once loaded, displaying a spinner until that point
+- react-native-bpk-component-text: 0.0.1-alpha.1
```

Comment:
The `react-native-bpk-component-text` version should be left blank in the changelog (We'll publish it at 1.0.0 for first proper release anyways).

Reply:
We'll drop the alpha comment too ;).

Reply:
Done